api/api.go

```diff
@@ -271,6 +272,21 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if silencedParam := r.FormValue("silenced"); silencedParam != "" {
```

Comment:
We'd commit to accepting all true/false variations `ParseBool` provides. Keeping it simple for now is a safe bet.